src/components/BrowserExamplesMenu/snippets/dispatch.js

```diff
@@ -29,7 +29,7 @@ export default async function() {
 
     const h3 = page.locator("h3");
     const ok = await h3.textContent() == "Contact us";
-    check(ok, { "header": ok });
+    await check(ok, { "header": ok });
```

Comment:
I think we should change some of these snippets to match the intent of the async/await so it is clearer what it is doing:

```
    await check(page.locator("h3"), {
      header: async (lo) => (await lo.textContent()) === "Contact us",
    })
```